tests/integration-tests/tests/spot/test_spot.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutor
+from tests.common.schedulers_common import get_scheduler_commands
+
+
+@pytest.mark.regions(["eu-west-1"])
```

Comment:
since this is spot let's pick one of the biggest regions such as us-east-1?

Reply:
OK